notebook/i18n/fr_FR/LC_MESSAGES/nbui.po

```diff
@@ -0,0 +1,746 @@
+msgstr "Connexion non disponible, vous ne devriez pas voir cette page."
+
+#: notebook/templates/logout.html:24
+#, python-format
+msgid "Proceed to the <a href=\"%(base_url)s\">dashboard"
+msgstr "Continuer vers le <a href=\"%(base_url)s\">tableau de bord"
+
+#: notebook/templates/logout.html:26
+#, python-format
+msgid "Proceed to the <a href=\"%(base_url)slogin\">login page"
+msgstr "Continuer vers la <a href=\"%(base_url)s\">page de connexion"
+
+#: notebook/templates/notebook.html:62
+msgid "Menu"
+msgstr "Menu"
+
+#: notebook/templates/notebook.html:65 notebook/templates/notebook.html:254
+msgid "Kernel"
+msgstr "Noyau"
+
+#: notebook/templates/notebook.html:68
+msgid "This notebook is read-only"
+msgstr "Ce notebook est en lecture seule"
+
+#: notebook/templates/notebook.html:81
+msgid "New Notebook"
+msgstr "Nouveau Notebook"
+
+#: notebook/templates/notebook.html:85
+msgid "Opens a new window with the Dashboard view"
+msgstr "Ouvre une nouvelle fenêtre de tableau de bord"
+
+#: notebook/templates/notebook.html:86
+msgid "Open..."
+msgstr "Ouvrir…"
+
+#: notebook/templates/notebook.html:90
+msgid "Open a copy of this notebook's contents and start a new kernel"
+msgstr "Ouvre une copie du contenu de ce notebook et démarre un nouveau noyau"
+
+#: notebook/templates/notebook.html:91
+msgid "Make a Copy..."
+msgstr "Créer une Copie…"
+
+#: notebook/templates/notebook.html:92
+msgid "Rename..."
+msgstr "Renommer…"
+
+#: notebook/templates/notebook.html:93
+msgid "Save and Checkpoint"
+msgstr "Créer une nouvelle sauvegarde"
+
+#: notebook/templates/notebook.html:96
+msgid "Revert to Checkpoint"
+msgstr "Restaurer une Sauvegarde"
+
+#: notebook/templates/notebook.html:106
+msgid "Print Preview"
+msgstr "Imprimer l'Aperçu"
+
+#: notebook/templates/notebook.html:107
+msgid "Download as"
+msgstr "Télécharger au format"
+
+#: notebook/templates/notebook.html:109
+msgid "Notebook (.ipynb)"
+msgstr "Notebook (.ipynb)"
+
+#: notebook/templates/notebook.html:110
+msgid "Script"
+msgstr "Script"
+
+#: notebook/templates/notebook.html:111
+msgid "HTML (.html)"
+msgstr "HTML (.html)"
+
+#: notebook/templates/notebook.html:112
+msgid "Markdown (.md)"
+msgstr "Markdown (.md)"
+
+#: notebook/templates/notebook.html:113
+msgid "reST (.rst)"
+msgstr "reST (.rst)"
+
+#: notebook/templates/notebook.html:114
+msgid "LaTeX (.tex)"
+msgstr "LaTeX (.tex)"
+
+#: notebook/templates/notebook.html:115
+msgid "PDF via LaTeX (.pdf)"
+msgstr "PDF via LaTeX (.pdf)"
+
+#: notebook/templates/notebook.html:118
+msgid "Deploy as"
+msgstr "Déployer en tant que"
+
+#: notebook/templates/notebook.html:123
+msgid "Trust the output of this notebook"
+msgstr "Faire confiance à la sortie de ce notebook"
+
+#: notebook/templates/notebook.html:124
+msgid "Trust Notebook"
+msgstr "Faire confiance au Notebook"
+
+#: notebook/templates/notebook.html:127
+msgid "Shutdown this notebook's kernel, and close this window"
+msgstr "Éteindre le noyau de ce notebook et fermer cette fenêtre"
```

Comment:
Éteindre ou Arrêter ? 
Les deux me vont, mais je sais que traduire est parfois dure. Up to you.

Reply:
Arrêter me semble mieux, j'harmonise.